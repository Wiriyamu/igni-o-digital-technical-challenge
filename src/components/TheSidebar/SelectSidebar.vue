<template>
  <div class="select">
    <select class="select__select">
      <option>√Årea do cliente</option>
    </select>
    <img class="select__icon" src="@/assets/images/home.svg" alt="Home" />
  </div>
</template>

<script>
export default {
  name: 'SelectSidebar'
}
</script>

<style lang="scss" scoped>
@import '@/assets/sass/abstracts/_variables.scss';

.select {
  position: relative;

  @media screen and (max-width: 800px) {
    display: flex;
    justify-content: center;
  }

  &__select {
    max-width: 23rem;
    width: 100%;
    height: 3.6rem;
    margin: 0 2rem;
    padding: 0.3125rem 0.3125rem 0 3rem;
    border: 0;
    border-radius: 0.3125rem;
    background-color: $selectSidebarBg;

    @media screen and (max-width: 800px) {
      padding: 0.3125rem 0.3125rem;
    }
  }

  &__icon {
    position: absolute;
    top: 0.8rem;
    left: 3rem;

    @media screen and (max-width: 800px) {
      display: none;
    }
  }
}
</style>
