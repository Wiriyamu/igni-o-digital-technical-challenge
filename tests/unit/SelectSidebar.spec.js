import SelectSidebar from '@/components/TheSidebar/SelectSidebar'
import { mount } from '@vue/test-utils'

describe('SelectSidebar.vue', () => {
  it('Snapshot component', () => {
    const wrapper = mount(SelectSidebar)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
