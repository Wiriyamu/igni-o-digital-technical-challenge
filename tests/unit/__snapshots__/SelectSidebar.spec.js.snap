// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectSidebar.vue Snapshot component 1`] = `
<div class="select"><select class="select__select">
    <option>√Årea do cliente</option>
  </select><img class="select__icon" src="" alt="Home"></div>
`;
